cloto/constants.py

```diff
@@ -52,3 +54,8 @@
 #KEYSTONE_PROXY
 TOKEN_NOT_FOUND = u'User token not found'
 SERVICE_NOT_AUTORIZED = u'Service not authorized'
+DEFAULT_REQUEST_TIMEOUT = 60
+HTTP_RESPONSE_CODE_OK = 200
+HTTP_RESPONSE_CODE_UNAUTHORIZED = 401
```

Comment:
For a next release, you could use [httplib](https://docs.python.org/2/library/httplib.html#httplib.HTTPS_PORT) constants. Anyway, it is OK.